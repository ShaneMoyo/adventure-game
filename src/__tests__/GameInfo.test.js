import React from 'react';
import { shallow, mount } from 'enzyme';
import toJSON from 'enzyme-to-json';
import GameInfo from '../GameInfo';
import SelectBox from '../SelectBox';
import floors from '../Floors';


describe('App', () => {

  it('should render game info component', () => {
    const wrapper = shallow(<GameInfo holding="sauce" pizzaIngredients={['cheese', 'crust']} />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });

  it('should render SelectBox component', () => {
    const wrapper = shallow(<SelectBox options={floors} className="dropDown one" name="elevator"
      selectChange={floorNumber => floorNumber} />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });


});