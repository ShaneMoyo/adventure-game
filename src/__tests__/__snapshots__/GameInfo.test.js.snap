// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render SelectBoxu component 1`] = `
<select
  className="dropDown one"
  name="elevator"
  onChange={[Function]}
>
  <option
    key="0"
    value={0}
  >
    Ground Floor
  </option>
  <option
    key="1"
    value={1}
  >
    1st Floor
  </option>
  <option
    key="2"
    value={2}
  >
    2nd Floor
  </option>
  <option
    default={true}
    key="3"
    value={3}
  >
    Home
  </option>
  <option
    key="4"
    value={4}
  >
    4th Floor
  </option>
  <option
    key="5"
    value={5}
  >
    5th Floor
  </option>
  <option
    key="6"
    value={6}
  >
    6th Floor
  </option>
</select>
`;

exports[`App should render game info component 1`] = `
<div>
  <label>
     
  </label>
  <p>
    Holding: 
    sauce
  </p>
  <p>
    Pizza: 
    cheese,crust
  </p>
</div>
`;
